app/containers/Login/lib.js

```diff
@@ -0,0 +1,45 @@
+import Auth0Lock from 'auth0-lock';
+import { storeSecret } from 'components/AuthUser/lib';
+
+export const LOCK_CONTAINER_ID = 'lock-container';
+
+let lock;
+
+export function createAndShow(nextPathname) {
+  lock = createLock(nextPathname);
+  lock.show();
+}
+
+function createLock(nextPathname) {
+  const secret = createNonce();
+  storeSecret(secret);
+  return new Auth0Lock(process.env.AUTH0_CLIENT_ID, process.env.AUTH0_DOMAIN, {
+    auth: {
+      redirectUrl: `${window.location.origin}/login/callback`,
```

Comment:
I guess `window.location.origin` works only in chrome and FF not IE. It needs a polyfill:
````
if (!window.location.origin) {
  window.location.origin = window.location.protocol + "//" 
    + window.location.hostname 
    + (window.location.port ? ':' + window.location.port : '');
}
````